src/main.py

```diff
@@ -6,17 +6,23 @@
 import google.oauth2.id_token
 import requests
 from discord import Embed
+from discord.abc import GuildChannel
 from discord.ext.commands import Bot, Context
+from discord.member import Member as member_type
 from discord.message import Message as message_type
 from dotenv import load_dotenv
 from google.auth.transport.requests import Request
+from google.cloud.pubsub_v1 import PublisherClient
+from google.cloud.pubsub_v1.publisher.futures import Future
 from requests import Response
 
 load_dotenv()
 
 LOGGER: logging.Logger = logging.getLogger(__name__)
 DISCORD_API_TOKEN = os.getenv("DISCORD_API_TOKEN")
 FUNCTION_BASE_RUL = "https://us-central1-archy-f06ed.cloudfunctions.net/"
+PROJECT_ID = "archy-f06ed"
+TOPIC_ID = "welcome_new_user"
```

Comment:
Move this to the `on_member_join` function, we might need other Topic ID in this file, this is not a global var

Reply:
Ok, it will be done.